requirements.txt

```diff
@@ -1,2 +1,3 @@
 torch>=1.9
 numpy>=1.20
+nvidia-ml-py
```

Comment:
I think this should be an optional dependency. if it's not installed properly, we can simply print the warning message assuming `major >= 8` is true..